data/settings/icon-grid-C.json.in

```diff
@@ -53,11 +54,17 @@
   ],
   "eos-folder-games-to-hack.directory": [
     "com.endlessnetwork.aqueducts.desktop",
+    "eos-aqueducts.desktop",
```

Comment:
This is not going to work because on systems where Aqueducts is pre-installed, this would show both the real launcher and the Get launcher. You'll have to patch them into the image-specific icon grid layout in the image builder, I think. :(